test-harness/specs/validate-query-params/query-param-form-exploded-object-invalid.oas3.txt

```diff
@@ -0,0 +1,37 @@
+====test====
+When I send a request to an operation
+And the operation has a required exploded form query param
+And in the request I sent the object properties with values with invalid types (strings not integers)
+Then I should get a 422 response back
+====spec====
+openapi: '3.0.2'
+paths:
+  /recipes:
+    get:
+      parameters:
+        - name: pagination
+          in: query
+          required: true
+          style: form
+          explode: true
+          schema:
+            type: object
+            properties:
+              offset:
+                type: integer
+                example: 0
+              limit:
+                type: integer
+                example: 25
+      responses:
+        '200':
+          description: OK
+====server====
+mock -p 4010 ${document}
+====command====
+curl -i -X GET http://127.0.0.1:4010/recipes\?limit\=string\&offset\=string
```

Comment:
why escaping here and not other places?

Reply:
I'm not entirely sure... without the escaping, I wasn't getting both parameter validations, I was only getting offset. And this is how my terminal handled it when I pasted the url in there, and so I copied the escaping over and it got both validations to come across in the test 🤷‍♀️